lms/djangoapps/bulk_email/migrations/0003_multiple_choices_to_option.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import util.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('bulk_email', '0002_data__load_course_email_template'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='courseemail',
+            name='to_option',
+            field=util.models.SerializedChoicesField(default=[b'myself'], max_length=64, choices=[(b'myself', b'Myself'), (b'staff', b'Staff and instructors'), (b'learners', b'All students')]),
+        ),
+    ]
```

Comment:
Because of how I designed `SerialChoicesField`, this migration becomes a nullop, and all previous data is still valid.
```
edxapp@precise64:~/edx-platform$ ./manage.py lms --settings=aws sqlmigrate bulk_email 0003
2016-04-29 15:54:39,215 INFO 15672 [dd.dogapi] dog_stats_api.py:66 - Initializing dog api to use statsd: localhost, 8125
System check identified some issues:

WARNINGS:
wiki.ArticleRevision.ip_address: (fields.W900) IPAddressField has been deprecated. Support for it (except in historical migrations) will be removed in Django 1.9.
	HINT: Use GenericIPAddressField instead.
```